cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1213,6 +1214,21 @@ func checkDataAndSavePairing(rootdir string) (bool, error) {
 	return subtle.ConstantTimeCompare(marker1, marker2) == 1, nil
 }
 
+// waitPartSrc waits for the given partSrc to appear.
+//
+// No checks beyond file existance are performed. It can be mocked in
+// tests.
+var waitPartSrc = func(partSrc string, wait time.Duration, n int) error {
```

Comment:
nitpick: you seem to have named this function after the `partSrc` local variable used in `mountNonDataPartitionMatchingKernelDisk()`, but once we look at this function in isolation, the name becomes a bit confusing. :-)

IMHO `waitForFile()`, `waitUntilFileExists()`, `waitUntilFileAppears()` or similar names which describe what the function do (instead of describing a specific instance of its usage) would make the code more readable.

It's still a nitpick, so do the change only if you have to change the branch for other reasons :-)

Reply:
I called it just `waitFile()` now but happy to tweak as needed.